macos/FluentUI/Button/Button.swift

```diff
@@ -598,15 +613,16 @@ private struct ButtonSizeParameters {
 		titleToImageSpacing: 10,
 		titleToImageVerticalSpacingAdjustment: 7
 	)
-	
+
 	static let medium = ButtonSizeParameters(
 		fontSize: 13,  // line height: 17
 		cornerRadius: 6,
-		verticalPadding: 2.0,  // overall height: 24
+		verticalPadding: 2.0, // overall height: 24
 		horizontalPadding: 5,
 		titleVerticalPositionAdjustment: 0,
 		titleToImageSpacing: 3,
-		titleToImageVerticalSpacingAdjustment: 7
+		titleToImageVerticalSpacingAdjustment: 7,
+		minButtonHeight: 24
```

Comment:
Yeah, per the other comment also adding minButtonHeight values for small and large seems worth it as the risk of those buttons changing a few pixels taller would be small and arguably by design since the sizes are already mentioned in the structs.